src/serve.rs

```diff
@@ -4,8 +4,13 @@ use std::sync::Arc;
 use anyhow::Result;
 use async_std::fs;
 use async_std::task::{spawn, spawn_local, JoinHandle};
+use futures::channel::mpsc::{channel, Receiver};
+use futures::lock::Mutex;
+use futures::{StreamExt, TryFutureExt};
```

Comment:
I wish we would have already cut over to tokio. Tokio's broadcast channels would be much better for this use case. A single broadcast sender from the WatchSystem. Any number of WebSocket connections could then be open and be subscribed to the signal.